packages/web_benchmarks/pubspec.yaml

```diff
@@ -20,6 +20,7 @@ dependencies:
   shelf: ^1.2.0
   shelf_static: ^1.1.0
   test: ^1.19.5
+  web: '>=0.3.0 <0.5.0'
```

Comment:
Why the `<0.5.0` constraint?

Reply:
`<1.0.0` would also capture `0.4.0`, right? And it would also be less sensitive to version constraints in adjacent packages.

Reply:
But we're not sure about what we're doing in `0.5.0+` This is a "good" range for things we've tested.